torchaudio/models/wav2letter.py

```diff
@@ -0,0 +1,71 @@
+import torch
+from torch import nn
+
+__all__ = ["Wav2Letter"]
+
+
+class Wav2Letter(nn.Module):
+    r"""Wav2Letter model architecture from the `"Wav2Letter: an End-to-End ConvNet-based Speech Recognition System"
+     <https://arxiv.org/abs/1609.03193>`_ paper.
+
+    Args:
+        num_classes (int, optional): Number of classes to be classified. (Default: ``40``)
+        version (str, optional): Wav2Letter can use as input: ``waveform``, ```power_spectrum``
+         or ```mfcc``. (Default: ``waveform``)
+        n_input_features (int, optional): If is used ``power_spectrum`` or ```mfcc`` must be
+        specified the number of features that were extracted. (Default: ``None``)
+    """
+
+    def __init__(self, num_classes=40, version="waveform", n_input_features=None):
+        super(Wav2Letter, self).__init__()
+        n_input_features = 250 if not n_input_features else n_input_features
```

Comment:
Can a user want to specify a number of feature different from 1 when using the waveform version?

Reply:
Yes, it might want two use two channels for example